apps/src/templates/sectionProgressV2/LessonDataCell.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {studentLessonProgressType} from '../progress/progressTypes';
+import classNames from 'classnames';
+import styles from './progress-table-v2.module.scss';
+import {lessonHasLevels} from '../progress/progressHelpers';
+
+export default function LessonDataCell({lesson, studentLessonProgress}) {
+  const lessonData = React.useMemo(() => {
+    if (!lessonHasLevels(lesson)) {
+      return 'nolev';
+    }
+    if (!studentLessonProgress) {
+      return;
+    }
+    return Math.round(studentLessonProgress.completedPercent) + '%';
+  }, [lesson, studentLessonProgress]);
```

Comment:
Just so I understand how this page over all will work, this will update when a user selects a different lesson (which makes sense), but will `studentLessonProgress` ever change unless the page is re-loaded?  That is, this page won't live update while students are working, correct?

Reply:
That's correct. This might change later, but just needed to load the data to make the grid. I have a separate task to make sure the data loading is correct [TEACH-803](https://codedotorg.atlassian.net/browse/TEACH-803)